api/pom.xml

```diff
@@ -88,6 +88,11 @@
          <artifactId>spring-context-support</artifactId>
       </dependency>
       <dependency>
+         <groupId>org.springframework.data</groupId>
+         <artifactId>spring-data-jpa</artifactId>
+         <version>1.9.1.RELEASE</version>
```

Comment:
Not the latest version,because of compatibility issue with Spring. The latest one requires Spring 4.2.8.RELEASE